js/stores/appStore.js

```diff
@@ -614,6 +619,12 @@ const handleAppAction = (action) => {
       appState = appState.setIn(['sync', 'devices'], {})
       appState = appState.setIn(['sync', 'objectsById'], {})
       break
+    case appConstants.APP_REMOVE_SYNC_DEVICE:
+      appState = appState.deleteIn(['sync', 'devices', action.deviceId])
```

Comment:
I don't know if this is the intended behavior, but since this does not delete the device seed, if you set up sync again after removing a device, it will re-join the old sync chain.

Reply:
this is probably not the right behavior because the text says 'Start a new sync chain' not 'Rejoin my old sync chain'

Reply:
correct. It should start new, not rejoin.

Reply:
not sure if I get it but isn't the seed related to the main device only?

Reply:
so that option is for other devices (other than the main device). IIRC considering the above use case, if user setup sync again after removing a device, `seed` will be cleared given that removing the main device triggers `RESET_SYNC`.

Reply:
@cezaraugusto 

i think tese were my test steps:
1. join an existing sync chain (the one corresponding to bytes `[0, 0, 0, 0...]`, so the current device is not the main device
2. click `Leave Sync Chain`
3. click `Start a new sync chain`
4. click `Computer`
5. it showed the old sync code

however i can't seem to repro this consistently

Reply:
sorry unable to repro but by clicking `Leave Sync Chain` you trigger `DELETE_SYNC_USER`which iirc clears all data in client. Maybe a response delay from the server? 

Created #13799 for further investigation